src/app/core/entity-components/entity-list/list-paginator/list-paginator.component.ts

```diff
@@ -99,7 +103,7 @@ export class ListPaginatorComponent<E extends Entity>
       this.paginatorPageSize = po.length > 2 ? po[po.length - 2] : po[0];
     }
     this.paginator._changePageSize(this.paginatorPageSize);
-    this.allToggle = !this.allToggle;
```

Comment:
Why did you change this? For me this needs to be

this.allToggle = !this.allToggle;

instead of 

this.allToggle = this.paginatorPageSize >= this.dataSource.data.length;

for the component to work.